src/quo/components/wallet/wallet_activity/schema.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo.components.wallet.wallet-activity.schema
+  (:require [quo.components.tags.context-tag.schema :as ?context-tag]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:transaction {:optional true} [:maybe [:enum :receive :send :swap :bridge :buy :destroy :mint]]]
+      [:status {:optional true} [:maybe [:enum :pending :confirmed :finalised :failed]]]
+      [:counter {:optional true} [:maybe :int]]
+      [:timestamp {:optional true} [:maybe :string]]
+      [:blur? {:optional true} [:maybe :boolean]]
+      [:on-press {:optional true} [:maybe fn?]]
+      [:state {:optional true} [:maybe [:enum nil :disabled]]]
```

Comment:
`:maybe` already handles the `nil` case. If it can only be nil or `:disabled`, you can do 
```clj
[:state {:optional true} [:maybe [:= :disabled]]]
```

Reply:
Nice 👌 Done